torch/_dynamo/convert_frame.py

```diff
@@ -1059,6 +1060,13 @@ def format_guard_failures() -> str:
                 torch._logging.get_structured_logging_overhead()
             )
 
+            def convert_sets_to_strings(d: Dict[str, Any]) -> Dict[str, Any]:
+                return {
+                    key: str(value) if isinstance(value, set) else value
```

Comment:
Wouldn't a list be better here?

Reply:
Yes, updated to list.

Reply:
This is exactly why you need filtering! Don't just put garbage data in Scuba, it's not free

Reply:
Makes sense, added filtering to exclude sets  containing _TensorMeta and functions.